.github/workflows/CI.yml

```diff
@@ -263,6 +264,7 @@ jobs:
           USES_DOCKER_PGSQL: true
           ETH_ALCHEMY_API_KEY: ${{ secrets.ETH_ALCHEMY_API_KEY }}
           ENTITIES_URL: ${{ secrets.ENTITIES_URL }}
+          DATABASE_URL: postgresql://commonwealth:edgeware@localhost/commonwealth # required in production
```

Comment:
Why do we need this? I added `TEST_DB_NAME` to bypass `NODE_ENV=production` requiring `DATABASE_URL` to be defined.

Reply:
too many rules to remember, better to enforce required vars in prod mode

Reply:
In that case we should remove the `TEST_DB_NAME` env var from the command.